code/game/objects/items/weapons/grenades/flashbang.dm

```diff
@@ -6,11 +6,30 @@
 
 /obj/item/grenade/flashbang/prime()
 	..()
+	/// Holding a flashbang when it goes off is bad news.
```

Comment:
These aren't DMDocs, double slash only goes here, DMDocs are just for vars and procs

```suggestion
	// Holding a flashbang when it goes off is bad news.
```